src/Core/src/Primitives/LayoutAlignment.cs

```diff
@@ -1,30 +1,33 @@
-﻿namespace Microsoft.Maui.Primitives
+﻿using System;
+
+namespace Microsoft.Maui.Primitives
 {
 	// We don't use Microsoft.Maui.Controls.LayoutAlignment directly because it has a Flags attribute, which we do not want
 
 	/// <summary>
 	/// Determines the position and size of an IView when arranged in a parent element
 	/// </summary>
+	[Flags]
```

Comment:
the comment says, literally:
```
// We don't use Microsoft.Maui.Controls.LayoutAlignment directly because it has a Flags attribute, which we do not want
```

Reply:
I missed this comment, updated.